orbit/pkg/table/extension_darwin.go

```diff
@@ -61,6 +62,7 @@ func PlatformTables(opts PluginOpts) ([]osquery.OsqueryPlugin, error) {
 		table.NewPlugin("corestorage_logical_volume_families", corestorage.LogicalVolumeFamiliesColumns(), corestorage.LogicalVolumeFamiliesGenerate),
 		table.NewPlugin("filevault_prk", filevault_prk.Columns(), filevault_prk.Generate),
 		table.NewPlugin("find_cmd", find_cmd.Columns(), find_cmd.Generate),
+		table.NewPlugin("macos_user_profiles", macos_user_profiles.Columns(), macos_user_profiles.Generate),
```

Comment:
Do I have to manually update `osquery_fleet_schema.json` and `fleed_logs.yml` in `/schema/`? And `mode-fleet-highlight-rules.js`? To check with Lucas, probably.

Reply:
You need to create a `.yml` file in `schema/tables/` and then auto-generate the `osquery_fleet_schema.json` file. This is how:
https://github.com/fleetdm/fleet/blob/bc08109ff1c5b1ae9a00fb44d508a6bab25df73a/.github/workflows/check-automated-doc.yml#L59-L61